docs/docusaurus/static/_redirects

```diff
@@ -443,4 +443,21 @@ docs/guides/setup/configuring_data_docs/how_to_host_and_share_data_docs_on_azure
 
 /docs/reference/expectations/standard_arguments/* /docs/reference/learn/expectations/standard_arguments/:splat
 /docs/reference/expectations/* /docs/reference/learn/expectations/:splat
-/docs/reference/terms/* /docs/reference/learn/terms/:splat
\ No newline at end of file
+/docs/reference/terms/* /docs/reference/learn/terms/:splat
+
+
+# Redirects for GX Core prerelease
+/docs/oss/guides/expectations/creating_custom_expectations/*    /docs/core/expectations/manage_expectations#customize-an-expectation-class
+/docs/oss/guides/expectations/how_to_edit_an_existing_expectationsuite  /docs/core/expectation_suites/manage_expectation_suites
+/docs/oss/guides/expectations/* /docs/core/expectations/manage_expectations
+
+/docs/oss/guides/expectations/how_to_create_and_edit_expectations_based_on_domain_knowledge_without_inspecting_data_directly
+/docs/oss/guides/expectations/how_to_create_and_edit_expectations_with_instant_feedback_from_a_sample_batch_of_data
+
```

Comment:
My intention is to add more granular redirects as additional docs are created and existing docs are revised/consolidated.  The more granular redirects will be placed above these broad ones and will therefore take priority: our redirects system finds the first valid match and follows that rule, so the general ones will only be in place to catch files we may have overlooked and send people to a reasonable starting point for finding what they want.

That said, these redirects are going to need to be revised based on the path we settle on for the final "prerelease" version, so when we do cut that version I'm going to advise we drop the changes to `_redirects` and to `docusaurus.config.js` since they won't reflect the behavior we want after the prerelease version has been cut.